scalableminds/webknossos
# Fix statistics overview graph error

This PR fixes the occurrence of an error when clicking on the graph of the statistics overview page.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to statistics > overview
- click on a point of the Overall Weekly Annotation Time (hours per week) graph 
- there should be no error

### Issues:
- fixes #4779

------
